.azure/pipelines/jobs/codesign-xplat.yml

```diff
@@ -26,6 +26,7 @@ jobs:
       inputs:
         solution: eng\tools\XplatPackageSigner\XplatPackageSigner.proj
         msbuildArguments: /p:SignType=$(_SignType)
+          /p:Configuration=Release
           -p:AssetManifestFileName=aspnetcore-${{ parameters.inputName }}-signed.xml
```

Comment:
Also change this to `/p:` because `-p` sometimes confuses our build scripts and MSBuild on Windows.

Reply:
Separately, docs for the publishing feature are found at https://github.com/dotnet/arcade/blob/master/Documentation/CorePackages/Publishing.md and the main entry point is in https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj

A few possibilities since the `$Configuration)` change probably won't fix the fact no publication is occurring, I suggest you try:
1. Debug the binary log to see whether the properties on the command line aren't overridden or lost somewhere in the `msbuild` world. Fix anything you find.
2. Add `-restore` to the command line i.e. the Arcade SDK might not be imported
3. Hack something together which creates the artifacts\ folder and maybe a few subfolders (if they don't exist for some reason)
4. Add `/verbosity:minimal` or even `/verbosity:quiet` to the command line. This seems like a longshot given how small the console log for this job is. But, @JohnTortugo suggested it because this helped in another repo.